docs/src/modules/utils/getProductInfoFromUrl.ts

```diff
@@ -51,7 +51,17 @@ export default function getProductInfoFromUrl(asPath: string): MuiProductInfo {
     }
   }
 
-  if (firstFolder === 'toolpad' || firstFolder === 'docs') {
+  if (firstFolder === 'toolpad') {
+    productCategoryId = 'toolpad';
+    const secondFolder = asPathWithoutLang.replace(/^\/+[^/]+\/([^/]+)\/.*/, '$1');
+    if (secondFolder === 'studio') {
+      productId = 'toolpad-studio';
+    } else {
+      productId = 'toolpad-core';
+    }
+  }
```

Comment:
Can we fix `MuiProductId` and `MuiProductCategoryId`? These types are wrong now. Thanks